src/MoviesList/index.js

```diff
@@ -38,11 +39,10 @@ class MoviesList extends Component {
 
     render() {
         const {data} = this.props;
-        const getFullYear = new Date()
         const MovieList = () => data.map(movie =>
```

Comment:
you shouldn't do it in render function, it's your `MovieList` component, that should be defined independently

plus, as for me Search, Sort and other staff is part of the list (but separated by smaller components and commbined in one with container)

Reply:
This was extra - background is red. Yes, I wrote it initial wrong. Defined a separate method.

getYear(val) {
       return new Date(val).getFullYear();
   }